Lesson24-signup-form/assets/assets/ezgi/styles.css

```diff
@@ -0,0 +1,143 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  body {
+    font-family: Arial, sans-serif;
+    min-width:320px ;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #3c3952;
+  }
+   #confirm{
+    display: none;
+    background-color: #fff;
+  } 
+
+  .confirm-active{
+border: 1px solid black;
+border-radius: 20px;
+font-weight:lighter;
+    width: 400px;
+    text-align: left;
+    padding: 20px;
+  } 
+
+
+  
+  #container{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 800px;
+    background-color: #fff;
+    padding: 15px;
+    border-radius: 8px;
+  }
+  
+  
+  .form-container {
+    width: 350px;
+  }
+  
+  h1 {
+    font-size:45px;
+  }
+
+  p, ul {
+    color: #1f1f1f;
+  }
+
+  #email ,  .submit-button {
+    padding: 15px;
+    width: 90%;
+    border-radius: 6px;
+}
+
+  p{
+    width: 90%;
+    font-size: 16px;
+
+  }
+
+  ul{
+    padding-left: 8px ;
+    font-size: 17px;
+    margin-bottom: 30px;
+  }
+
+  li{
+    margin-bottom: 8px;
+  }
+
+  .email-top{
+    display: flex;
+    justify-content: space-between;
+    font-size: 12px  ;
+  }
+
+  .form-label , #emailInput {
+    font-weight:bold;
+  }
+  
+  #email {
+    margin-bottom: 20px;
+    border: 1px solid;
+    color: #0b0b0c;
+    border-color: #0b0b0c;
+  }
+  
+  .error {
+    color: #590808;
+    margin-bottom: 10px;
+    padding: 0;
+  }
+  
+
+  .submit-button {
+    background-color: #090334;
+    color: #fff;
+    border: none;
+    cursor: pointer;
+  }
+
+  .submit-button:hover {
+    background-color: #160884;
+  }
+
+
+
+  .img-mobile{
+    display: none;
+  }
+
+
+
+  @media only screen and (max-width: 600px) {
+    /* For mobile phones: */
+    #container{
+        flex-wrap: wrap-reverse;    
+        align-content:space-between;
+        padding-top: 0;
+        width: 100%;
+        height: 100vh;
+    }
+
+    .form-container {
+        margin: 20px  ;
```

Comment:
Add `flex:1;` to make the element take entire width available.
Don't add margin on sides, only on top: `margin-top: 20px;`